pkg/storaged/client.js

```diff
@@ -621,6 +623,35 @@ client.stop_mount_users = (users) => {
         return Promise.resolve();
 };
 
+/* Direct mounting and unmounting
+ *
+ * We don't use UDisks2 for most of our mounting and unmounting in
+ * order to get better control over which entry from fstab is
+ * selected.  Once UDisks2 allows that control, we can switch back to
+ * it.
+ *
+ * But note that these functions still require there to be an entry in
+ * fstab.
+ */
+
+client.mount_at = (block, target) => {
+    const entry = utils.array_find(block.Configuration,
+                                   c => c[0] == "fstab" && utils.decode_filename(c[1].dir.v) == target);
+    if (entry)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-16991-20220830-101236-2e5e4dfa-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)